prototype/_sass/libraries/patterns/components/_modal.scss

```diff
@@ -103,9 +103,9 @@ div.pat-modal {
     z-index: 2;
     border: 1px #dedede solid;
     box-shadow: 0 0.1em 0.4em rgba(0,0,0,0.2);
-    -moz-pointer-events:none;  
-    -webkkt-pointer-events:none;  
-    pointer-events:none;  
+    -moz-pointer-events:none;
+    -webkkt-pointer-events:none;
```

Comment:
I wonder how long it would have taken us to spot that if my editor had not stripped the whitespace automatically :)